api/api_autogen/library/defaults/Pvsamv1_FlatPlatePVLCOECalculator.json

```diff
@@ -182,24 +182,24 @@
 			"cec_v_oc_ref": 64.4
 		},
 		"CECPerformanceModelWithUserEnteredSpecifications": {
-			"6par_aisc": 0.004,
```

Comment:
It looks like these are still 6par instead of sixpar on develop (and pysam-v2.2.1) in ssc. Is this going to break anything?

Reply:
No it still works because the `marshal` module in Python automatically converts it to "six" when creating the default `.df` files

Reply:
Nevermind, I misinterpreted this. The problem is that the `6par_aisc` name as a pysam variable never worked. In PySAM we have to rename it so that it doesn't start with a number. In SSC, we could change everything, but that would be really messy. The part that we could do better is in the PySAM JSON Code Generator.